_data/radar.yaml

```diff
@@ -1,13 +1,171 @@
+# ring 0 = ADOPT
+# ring 1 = TRIAL
+# ring 2 = ASSESS
+# ring 3 = HOLD
+
 version: 2019-11
 quadrants:
   - name: Security
-    technologies: []
+    technologies:
+
+      - name: Vault
+        link: https://www.vaultproject.io/
+        ring: 0
+
+      - name: Open Policy Agent
+        link: https://www.openpolicyagent.org/
+        ring: 1
 
   - name: CI/CD
-    technologies: []
+    technologies:
+
+      - name: Terraform
+        link: https://terraform.io
+        ring: 0
+
+      - name: Helm
+        link: https://helm.sh
+        ring: 0
+
+      - name: Jenkins X
+        link: https://jenkins-x.io/
+        ring: 1
+
+      - name: Flux
+        link: github.com/fluxcd/flux
+        ring: 1
+
+      - name: Tekton
+        link: https://tekton.dev
+        ring: 2
+
+      - name: GitHub Actions
+        link: https://github.com/features/actions
+        ring: 2
+
+      - name: Argo
+        link: https://argoproj.github.io/
+        ring: 2
+
+      - name: Bamboo
+        link: atlassian.com/software/bamboo
+        ring: 4
+
+      - name: Octopus Deploy
+        link: https://octopus.com/
+        ring: 3
+
+      - name: Travis CI
+        link: https://travis-ci.com/
+        ring: 3
+
+      - name: Jenkins
+        link: https://jenkins.io/
+        ring: 4
```

Comment:
Moved this to 4, discuss if you disagree.

Reply:
Agreed!